.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+## Description
+
+
+## Related Issue \#
```

Comment:
nit: what is the `\#` usage in Markdown?

Reply:
Since `#` is considered a special character in markdown, this is just escaping that character to ensure that the markdown parser ignores it and displays it as-is.